lib/modules/platform/github/index.ts

```diff
@@ -1731,6 +1732,28 @@ export async function updatePr({
   }
 }
 
+export async function reattemptPlatformAutomerge({
+  prNo,
+  platformOptions,
+}: ReattemptPlatformAutomergeConfig): Promise<void> {
+  try {
+    const { body: ghPr } = await githubApi.getJson<GhRestPr>(
+      `repos/${config.parentRepo ?? config.repository}/pulls/${prNo}`,
+    );
+    const result = coerceRestPr(ghPr);
+    const { number, node_id } = result;
```

Comment:
I think you should fetch the PR from cache, so we don't need this extra API call.

@zharinov do we've the `node_id` on PR cache?

Reply:
Looks like it's not on the type, at least.

> lib/modules/platform/github/index.ts:1741:21 - error TS2339: Property 'node_id' does not exist on type 'Promise<GhPr | null>'.

I'll poke at this a bit